fastify.d.ts

```diff
@@ -102,6 +103,7 @@ export type FastifyServerOptions<
   onProtoPoisoning?: ProtoAction,
   onConstructorPoisoning?: ConstructorAction,
   logger?: boolean | FastifyLoggerOptions<RawServer> | Logger,
+  serializerOpts?: FJSOptions,
```

Comment:
@Eomm could this be customized somehow?

Reply:
Yes, users who implemented a custom [`compilersFactory.buildSerializer`](https://www.fastify.io/docs/latest/Server/#schemacontroller) may have other options

Reply:
How could we type that?

Reply:
There is not a standard interface to apply. To avoid `any` we could set `Record<string, unknown>`

Reply:
Updated to `FJSOptions | Record<string, unknown>`